src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php

```diff
@@ -412,6 +412,16 @@ protected function sendEarlyHints(iterable $links = [], ?Response $response = nu
         return $response;
     }
 
+    /**
+     * Call FormInterface::handleRequest() and return the result of the condition FormInterface::isSubmitted() and FormInterface::isValid().
+     */
+    protected function isSubmittedFormValid(FormInterface $form, Request $request): bool
+    {
+        $form->handleRequest($request);
```

Comment:
we could remove the $request argument and use the request_stack service to retrieve it instead, WDYT?